aws_lambda_events/src/apigw/mod.rs

```diff
@@ -587,6 +587,7 @@ pub struct ApiGatewayCustomAuthorizerRequestTypeRequestContext {
     #[serde(deserialize_with = "deserialize_lambda_string")]
     #[serde(default)]
     pub request_id: Option<String>,
+    #[serde(default)]
     pub identity: ApiGatewayCustomAuthorizerRequestTypeRequestIdentity,
```

Comment:
I would prefer to make this field optional to follow the convention with the rest of the structures:
```suggestion
    pub identity: Option<ApiGatewayCustomAuthorizerRequestTypeRequestIdentity>,
```

Reply:
I agree, done